Animations/Rotate.js

```diff
@@ -0,0 +1,30 @@
+var makeAnimation = require('../src/TuxAnimations');
```

Comment:
Add 'use strict'; on line one and leave a newline between that and variable declarations.